superset-frontend/src/components/ListView/TableCollection.tsx

```diff
@@ -29,6 +30,18 @@ interface Props {
   rows: TableInstance['rows'];
   loading: boolean;
 }
+
+const TableCell = styled.td`
+  width: ${(props: { size?: Column['size'] }) => {
+    if (props.size === 'xs') return '25px';
+    if (props.size === 'sm') return '50px';
+    if (props.size === 'md') return '75px';
+    if (props.size === 'lg') return '100px';
+    if (props.size === 'xl') return '150px';
+    if (props.size === 'xxl') return '200px';
+    return '';
+  }};
```

Comment:
this could be simpler if you have each if only return a number, then you append `px` on at the end. Note the final return would need to return 0 instead of empty string then.

on a side note, do you think a switch statement is better here? it would remove even more code duplication